usbreset.c

```diff
@@ -104,16 +109,13 @@ static void list_devices(void)
 		return;
 
 	while ((dev = parse_devlist(devs)) != NULL)
-		printf("  Number %03d/%03d  ID %04x:%04x  %s\n",
-			   dev->bus_num, dev->dev_num,
-			   dev->vendor_id, dev->product_id,
-			   dev->product_name);
+		printf("  Number %03d/%03d  ID %04x:%04x  %s %s\n", dev->bus_num, dev->dev_num, dev->vendor_id,
```

Comment:
Shouldn't you have 2 spaces after the string and before the serial number as it kind of looks like "SN:xxxx" is part of the device name.

Any chance of making the list a bit cleaner?

Reply:
Sure. Changed in a573189.